pkg/roachprod/install/expander.go

```diff
@@ -175,13 +181,25 @@ func (e *expander) maybeExpandPgHost(
 	if m == nil {
 		return s, false, nil
 	}
-
-	if e.pgHosts == nil {
-		var err error
-		e.pgHosts, err = c.pghosts(ctx, l, allNodes(len(c.VMs)))
+	switch m[1] {
+	case ":L":
+		addresses, err := c.ListLoadBalancers(l)
```

Comment:
Hrmm, don't the semantics of `pghost` get a bit murky with this implementation of `pghost`?

Say I create a load balancer for system and tenant in the cluster. What does it mean to say `{pghost:L}` in that case? Not sure if the `List*` functions are deterministic (w.r.t. ordering of LBs), in which case it could get even more confusing.

Maybe the "two load balancer" scenario is not common enough. It probably wouldn't hurt for `pghost` to take a virtual cluster name parameter as well.

Reply:
True, this won't work well for multiple load balancers. Adding a tenant group to this.